depends/packages/cppzmq.mk

```diff
@@ -0,0 +1,12 @@
+package=cppzmq
+$(package)_version=4.2.1
```

Comment:
looks like 4.2.2 is out: https://github.com/zeromq/cppzmq/tree/v4.2.2

Reply:
ACK, will add a commit to bump this dependency (and `zeromq` too, which also just release 4.2.3, so I'll try that and see if they are compatible, or otherwise just use 4.2.2).